engine/context/ecs-integration.md

```diff
@@ -86,7 +138,7 @@ arn:aws:secretsmanager:eu-west-3:12345:secret:DockerHubAccessToken
 
 Once created, you can use this ARN in you Compose file using using `x-aws-pull_credentials` custom extension with the Docker image URI for your service.
 
-```console
+```yaml
 version: 3.8

```

Comment:
unrelated: we should remove version here to promote version-less compose-spec

Reply:
Yep, thx, updated